workflows/Force_Password_Reset_in_Active_Directory_with_Teams/workflow.spec.yaml

```
@@ -0,0 +1,23 @@
+extension: workflow
+products: ["insightconnect"]
+name: Force_Password_Reset_in_Active_Directory_with_Teams
```

```suggestion
name: Force_Password_Reset_in_Active_Directory_with_Microsoft_Teams
```
Make sure to rename the workflow directory after making this change